src/main/java/seedu/address/storage/JsonAdaptedProperty.java

```diff
@@ -0,0 +1,143 @@
+package seedu.address.storage;
+
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.name.Name;
+import seedu.address.model.property.Address;
+import seedu.address.model.property.Deadline;
+import seedu.address.model.property.PostalCode;
+import seedu.address.model.property.Property;
+import seedu.address.model.property.Type;
+import seedu.address.model.property.client.Client;
+import seedu.address.model.remark.Remark;
+import seedu.address.model.util.DateTimeFormat;
+
+/**
+ * Jackson-friendly version of {@link Property}.
+ */
+class JsonAdaptedProperty {
+
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Appointment's %s field is missing!";
```

Comment:
Should `"Appointment's %s field is missing!"` be `"Property's %s field is missing!"`?

Reply:
Fixed, thanks!